architecture-baseline.json

```diff
@@ -27,6 +27,34 @@
         "ruleset": "Spryker",
         "priority": "2"
     },
+    {
+        "fileName": "/vendor/spryker-eco/payone/src/SprykerEco/Zed/Payone/Dependency/Facade/PayoneToGlossaryFacadeInterface.php",
+        "description": "\n      Bridges: The bridge interface has incorrect method 'createTranslation' signature. That violates the rule \"All bridge methods must have exactly the same signature as their interface\"\n    ",
```

Comment:
These can be fixed by using proper guidelined name for transfers

    LocaleTransfer $locale

=>

    LocaleTransfer $localeTransfer

Reply:
Then just run

    console c:s:a -m SprykerEco.Payone -u

again

Reply:
Fixed.